src/v/pandaproxy/schema_registry/types.h

```diff
@@ -23,6 +23,7 @@
 #include <seastar/util/bool_class.hh>
 
 #include <avro/ValidSchema.hh>
+#include <google/protobuf/descriptor.h>
```

Comment:
Please forward declare `google::protobuf::Descriptor` instead, but see the next comment.

Reply:
Took suggestion from other comment